[2022-09-28 05:34:20,810] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-09-28T05:00:17.130705+00:00 [queued]>
[2022-09-28 05:34:20,849] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-09-28T05:00:17.130705+00:00 [queued]>
[2022-09-28 05:34:20,850] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-28 05:34:20,851] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-09-28 05:34:20,851] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-28 05:34:20,913] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-09-28 05:00:17.130705+00:00
[2022-09-28 05:34:20,955] {standard_task_runner.py:52} INFO - Started process 418 to run task
[2022-09-28 05:34:20,980] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'manual__2022-09-28T05:00:17.130705+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpsi1_c8zz', '--error-file', '/tmp/tmprjw790ck']
[2022-09-28 05:34:20,990] {standard_task_runner.py:77} INFO - Job 224: Subtask local_to_gcs_task
[2022-09-28 05:34:21,426] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task manual__2022-09-28T05:00:17.130705+00:00 [running]> on host 7339e28b3b1f
[2022-09-28 05:34:21,769] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-09-28T05:00:17.130705+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-28T05:00:17.130705+00:00
[2022-09-28 05:34:33,888] {python.py:175} INFO - Done. Returned value was: None
[2022-09-28 05:34:33,927] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20220928T050017, start_date=20220928T053420, end_date=20220928T053433
[2022-09-28 05:34:33,995] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-28 05:34:34,038] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
